src/ChipInput.js

```diff
@@ -545,14 +569,16 @@ ChipInput.propTypes = {
   fullWidth: PropTypes.bool,
   fullWidthInput: PropTypes.bool,
   inputRef: PropTypes.func,
+  clearInputValueOnAdd: PropTypes.bool,
```

Comment:
This new prop is :+1:, but it should be documented in the readme. Also, the `propTypes` should be sorted alphabetically here.